features/step_definitions/dhis2/given.js

```diff
@@ -16,4 +16,17 @@ defineSupportCode(({ Given }) => {
       isVisible(dashboardPage.mainPageDiv);
     }
   );
+
+  Given(
+    /^I am authenticated$/,
+    () => {
+      // authenticate through api
+      login('admin', 'district');
+
+      browser.waitUntil(() => {
+        const url = browser.getUrl();
+        return url.indexOf('dhis-web') > -1;
```

Comment:
I think this might also give success in the case of a failed authentication. If you put in a bad password you are directed to e.g. https://play.dhis2.org/dev/dhis-web-commons/security/login.action?failed=true

Reply:
Good point! Can verify that login.action is not present

Reply:
:+1: